lib/clearance/configuration.rb

```diff
@@ -121,12 +121,20 @@ def  user_actions
       end
     end
 
+    # The name of user parameter for the configured user model.
+    # This is derived from the `model_name` of the `user_model` setting.
+    # In the default configuration, this is `user`.
+    # @return [Symbol]
+    def user_parameter
+      user_model.model_name.singular.to_sym
```

Comment:
This requires the user model to be an ActiveModel (or have `model_name`). I think that's reasonable, but is it already a requirement, or would this break backwards compatibility?